oak/proto/storage_channel.proto

```diff
@@ -16,73 +16,9 @@
 
 syntax = "proto3";
 
-package oak;
+package oak.storage;
 
-import "oak/proto/label.proto";
-
-// In all the request messages below, the storage_name field is used to partition data within an Oak
-// Module and is not exposed to the StorageProvider. The StorageChannel generates a unique storage
-// identifier from storage_name and the Oak Module identifier to prevent name collisions among Oak
-// Modules.
-
-message StorageItem {
-  bytes name = 1;
-  bytes value = 2;
-  oak.label.Label label = 3;
-}
-
-message StorageChannelReadRequest {
-  bytes storage_name = 1;
-  bytes transaction_id = 2;
-  StorageItem item = 3;
-}
-
-message StorageChannelReadResponse {
-  StorageItem item = 1;
-}
-
-message StorageChannelWriteRequest {
-  bytes storage_name = 1;
-  bytes transaction_id = 2;
-  StorageItem item = 3;
-}
-
-message StorageChannelWriteResponse {
-}
-
-message StorageChannelDeleteRequest {
-  bytes storage_name = 1;
-  bytes transaction_id = 2;
-  StorageItem item = 3;
-}
-
-message StorageChannelDeleteResponse {
-}
-
-message StorageChannelBeginRequest {
-  bytes storage_name = 1;
-}
-
-message StorageChannelBeginResponse {
-  bytes transaction_id = 1;
-}
-
-message StorageChannelCommitRequest {
-  bytes storage_name = 1;
-  bytes transaction_id = 2;
-}
-
-message StorageChannelCommitResponse {
-}
-
-message StorageChannelRollbackRequest {
-  bytes storage_name = 1;
-  bytes storage_id = 2;
-  bytes transaction_id = 3;
-}
-
-message StorageChannelRollbackResponse {
-}
+import "oak/proto/storage_messages.proto";
```

Comment:
Not sure I follow what's going on here. Why is the service definition not part of the ABI? It seems it should be.

Reply:
The generated code for the service definition assumes the use of the Oak SDK code, and so cannot live in the `oak_abi` crate.  That leads to the separation of the message definitions (which only depend on `prost` and so can live in `oak_abi`) from the service definition generated code (still in `oak`, but now with a parallel copy of the message definitions), which leads to my [earlier question](https://github.com/project-oak/oak/pull/834#issuecomment-611552052) &ndash; is it worth bothering with the storage protos?